creator-node/package.json

```diff
@@ -37,6 +37,7 @@
     "ffprobe-static": "^3.0.0",
     "ioredis": "^4.9.3",
     "ipfs-http-client": "^33.1.1",
+    "ipfs-http-client-cat": "npm:ipfs-http-client@^42.0.0",
     "ipfs-http-client-latest": "npm:ipfs-http-client@^42.0.0",
```

Comment:
why did we need two versions?

Reply:
:-) we dont...... I probably didn't add the npm uninstall .. changes. will add